src/App.svelte

```diff
@@ -102,7 +102,7 @@
           session={$session}
           activeRoute={$activeRouteStore} />
       {:else if $activeRouteStore.resource === "vesting"}
-        <Vesting {wallet} session={$session} activeRoute={$activeRouteStore} />
+        <Vesting {wallet} activeRoute={$activeRouteStore} />
```

Comment:
Do we even want to keep the special vesting route now that we can just use the search bar?

Reply:
Yeah I wanted to remove it initially but some core contributors received mails with the /vesting route, to enter/load their contract.
So didn't want to break that.
Maybe at some point in the future?

Reply:
Or we could just show instructions on that page that say: please paste your vesting contract into the search bar at the top?

Reply:
Yeah that doesn't sound like good UX.
IMO either:
- we have a page like currently where we only show vesting information under a specific route.
- or we add it to a profile page like this PR wants to do it, and:
  -  show it in some nice condensed graphical easy to understand way with graphs maybe?
  -  or we hide it with a dropdown to avoid having lots of lines of information.